h5c/vic/src/vic-webapp/src/app/vch-view/vch-log-view/vch-log-view.component.ts

```diff
@@ -34,41 +39,52 @@ export class VicVchLogViewComponent implements OnInit {
   constructor(
     private formBuilder: FormBuilder,
     private globalsService: GlobalsService,
-    private http: Http
+    private http: Http,
+    private createWzService: CreateVchWizardService,
+    private extSessionService: ExtendedUserSessionService
   ) {
     this.form = formBuilder.group({
       enableSSH: false
     });
   }
 
   ngOnInit() {
-    const vchId = this.vch.id.split(':')[3];
-    const serviceHost = '10.160.131.87';
-    const servicePort = '31337';
-    const targetHost = '10.192.93.240';
-    const targetThumbprint = this.globalsService.getWebPlatform().getUserSession()['serversInfo'][0]['thumbprint'];
+    this.loading = true;
+    Observable.zip(
+      this.createWzService.getVicApplianceIp(),
+      this.createWzService.acquireCloneTicket()
+    ).catch(err => {
+      console.error(err);
+      return Observable.throw(err);
+    }).subscribe(arr => {
+      const serviceHost = arr[0];
+      const cloneTicket = arr[1];
+      const vchId = this.vch.id.split(':')[3];
+      const servicePort = VIC_APPLIANCE_PORT;
+      const targetHost = this.extSessionService.getVcenterServersInfo()[0];
+      const targetHostname = targetHost.name;
```

Comment:
are we getting the vc name two different ways? I am going through the globalsService.getWebPlatform().getUserSession().serversInfo[0].name;

Reply:
So we have another service called `ExtendedUserSessionService` that basically includes what you did in your PR. (Sorry I should've let you know about this in the first place) It does pretty much the same